rest/models/src/main/java/com/connexta/store/rest/models/QuarantineRequest.java

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) Connexta
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package com.connexta.store.rest.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+/**
+ * Request
+ */
+public class QuarantineRequest {
```

Comment:
❗️This needs to include failure information

Reply:
I debated that. Is it not sufficient to expect polling to log that type of information for now with information is receiving from transformation?

Reply:
That is a good point, and we could leave it off for now.

I do think this is mixing responsibilities. I don't think the AttemptMaker should be responsible for dealing with failures at all. That should be the responsibility of the quarantine endpoint. The AttemptMaker should deal with the status, calling promote if completed or calling quarantine if the status is failed. You're right that right now, we only log something, but in the future we know 100% that we will need to do other things for different types of failures. These actions can only be done in the store service, so the store service would need the failure string.